client/core/core.go

```diff
@@ -4289,6 +4349,16 @@ func (c *Core) Login(pw []byte) error {
 	defer c.loginMtx.Unlock()
 
 	if !c.loggedIn {
+		// Derive the bond extended key from the seed.
+		seed, err := crypter.Decrypt(creds.EncSeed)
+		if err != nil {
+			return fmt.Errorf("seed decryption error: %w", err)
+		}
+		defer encode.ClearBytes(seed)
+		c.bondXPriv, err = deriveBondXPriv(seed)
+		if err != nil {
+			return fmt.Errorf("GenDeepChild error: %w", err)
+		}
```

Comment:
Should we not store this encrypted and decrypt it on demand? We could make the encrypted key part of the `PrimaryCredentials`.

Reply:
> decrypt it on demand

Not sure I see how that works.  In this scheme login is the "on demand" action.  When you login, bond maintenance begins, and it needs the keys at any moment.

Reply:
Oh right.